### PR TITLE
Adding wandb sweeps

### PR DESCRIPTION
**Pull request by tulasiram58827**

### CODE REVIEW

File: sweep.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+program: train.py
+method: bayes
+
+project: information_extraction
+
+metric:
+        name: test_pos_rec
```
Comment (radkoff) [inline sweep.yaml:7]: We should add an F1 metric and use that here. Otherwise, a model could just start to always predict positive labels and get good recall but bad precision (it is odd that on the latest run, precision never dips below 1.0; it's being very cautious about predicting positives)

File: train.py [hunk 2]
```diff
@@ -166,7 +176,7 @@ def train(dataset, test_data, n_instances, batch_size, epochs, model, optimizer,
                         help="Embedding dimension. Used for both token IDs and position")
     parser.add_argument('--attention_heads', type=int, default=8)
     parser.add_argument('--max_neighbors', type=int, default=30)
-    parser.add_argument('--epochs', type=int, default=100)
+    parser.add_argument('--epochs', type=int, default=10)
     parser.add_argument('--reports_per_epoch', type=int, default=4)
     parser.add_argument('--lr', type=float, default=0.001)
     parser.add_argument('--test_size', type=int, default=0.1)
```
Comment (radkoff) [inline train.py:182]: For the defaults here, refer to the dict above like `parser.add_argument('--epochs', type=int, default=hyperparameter_defaults['epochs'])`

File: sweep.yaml (hunk 1 above)
Comment (tulasiram58827) [reply sweep.yaml:7]: Right, Then I think we should also log F1 score also in train.py file.

Let me update that

File: train.py (hunk 2 above)
Comment (tulasiram58827) [reply train.py:182]: Cool
